modules/dnn/src/dnn.cpp

```diff
@@ -3488,7 +3488,8 @@ void Net::setParam(LayerId layer, int numParam, const Mat &blob)
     std::vector<Mat> &layerBlobs = ld.getLayerInstance()->blobs;
     CV_Assert(numParam < (int)layerBlobs.size());
     //we don't make strong checks, use this function carefully
-    layerBlobs[numParam] = blob;
+    blob.copyTo(layerBlobs[numParam]);
+    //layerBlobs[numParam] = blob;
```

Comment:
It is better to avoid unconditional duplication/copying if it is not necessary.

Try to use this instead if your application logic requires that:
```
net.setParam(... , my_mat.clone());
```

BTW, you may want to update documentation with hint.

Reply:
The problem is that it's Python sample and we try to set and use `cv::Mat` from numpy array which leads to segfault.

Reply:
Looks like with current state of the tracker we do not need this changes anymore.